viam-cartographer/src/carto_facade_queue/carto_facade_queue.go

```diff
@@ -0,0 +1,135 @@
+// Package cartofacadequeue defines all necessary components to enforce one call into c per cartofacade.
+package cartofacadequeue
```

Comment:
Lets change the directory structure a bit:

1. lets have all the code your created under a package called cartofacade
2. this file should be moved (with the same content) to cartofacade/queue.go